charger/eebus.go

```diff
@@ -505,7 +505,7 @@ func (c *EEBus) SoC() (float64, error) {
 	data, err := c.cc.GetData()
 	if err != nil {
 		c.log.TRACE.Printf("soc: no eebus data available yet")
```

Comment:
Fehlerbehandlung nur 1x: entweder loggen ODER zurück geben

Reply:
Ich habe das momentan mehrfach so gemacht, damit ich den Ablauf in den Trace Logs besser nachvollziehen kann und so schneller etwaige Probleme finde. Daher würde ich das so lange es `Experimental` ist gerne drin lassen.

Reply:
Ich hab die experimentellen Logs mit !! gestartet, die finde ich dan später schnell beim rausschmeissen. Wäre vllt auch für Dich eine Option